util/config.go

```diff
@@ -329,17 +334,44 @@ func (config *Config) getAndroidManifestPath() string {
 }
 
 func (config *Config) getAndroidConfigModelPath() string {
-	return path.Join(getPlatformSpecificAndroidClasspathPath(config), "OneginiConfigModel.java")
+	modelPath := path.Join(getPlatformSpecificAndroidClasspathPath(config), "OneginiConfigModel.java")
+	// if modelPath has no package name, check namespace property in build.gradle
+	if strings.HasSuffix(modelPath, "java/OneginiConfigModel.java") {
+		modelPath = strings.TrimSuffix(modelPath, "OneginiConfigModel.java")
+		modelPath = path.Join(modelPath, strings.ReplaceAll(config.getAndroidNamespacePath(), ".", "/"), "/OneginiConfigModel.java")
+	}
+	return modelPath
 }
 
 func (config *Config) getAndroidSecurityControllerPath() string {
-	return path.Join(getPlatformSpecificAndroidClasspathPath(config), "SecurityController.java")
+	modelPath := path.Join(getPlatformSpecificAndroidClasspathPath(config), "SecurityController.java")
```

Comment:
Same wrong path issue happens for security controller so I copy-pasted the implementation.